include/rovio/RovioInterfaceImplInl.hpp

```diff
@@ -377,6 +433,14 @@ bool RovioInterfaceImpl<FILTER>::getState(const bool get_feature_update,
     filter_update->qWI = state.poseRot(mpPoseUpdate_->inertialPoseIndex_);
   }
 
+  // Transformation between localization-map frame (G) and the odometry world
+  // frame (W).
+  if (mtState::enableMapLocalization_) {
+    filter_update->hasMapLocalizationPose = true;
```

Comment:
how do we know if it has one? is it general? or should it only be true when we actually localized?

Reply:
it is general. if the localization baseframe is part of the state it will be set to true (although there might be no localization updates).